appassembler-maven-plugin/pom.xml

```diff
@@ -252,6 +270,44 @@
           <include>wrapper-*</include>
         </includes>
       </resource>
+      <resource>
+        <directory>target/dependency/yajsw-binaries/yajsw-stable-12.14/bat</directory>
+        <targetPath>org/codehaus/mojo/appassembler/daemon/yajsw/bat</targetPath>
+      </resource>
+      <resource>
+        <directory>target/dependency/yajsw-binaries/yajsw-stable-12.14/bin</directory>
+        <targetPath>org/codehaus/mojo/appassembler/daemon/yajsw/bin</targetPath>
+      </resource>
+      <resource>
+        <directory>target/dependency/yajsw-binaries/yajsw-stable-12.14/scripts</directory>
+        <targetPath>org/codehaus/mojo/appassembler/daemon/yajsw/scripts</targetPath>
+        <includes>
+          <include>trayMessage.gv</include>
+        </includes>
+      </resource>
+      <resource>
+        <directory>target/dependency/yajsw-binaries/yajsw-stable-12.14/templates</directory>
+        <targetPath>org/codehaus/mojo/appassembler/daemon/yajsw/templates</targetPath>
+      </resource>
+      <resource>
+        <directory>target/dependency/yajsw-binaries/yajsw-stable-12.14/bin</directory>
+        <targetPath>org/codehaus/mojo/appassembler/daemon/yajsw/bin</targetPath>
+      </resource>
+      <resource>
+        <directory>target/dependency/yajsw-binaries/yajsw-stable-12.14/conf</directory>
+        <targetPath>org/codehaus/mojo/appassembler/daemon/yajsw/conf</targetPath>
+      </resource>
+      <resource>
+        <directory>target/dependency/yajsw-binaries/yajsw-stable-12.14/lib</directory>
+        <targetPath>org/codehaus/mojo/appassembler/daemon/yajsw/lib</targetPath>
+      </resource>
+      <resource>
+        <directory>target/dependency/yajsw-binaries/yajsw-stable-12.14/</directory>
+        <targetPath>org/codehaus/mojo/appassembler/daemon/yajsw/</targetPath>
```

Comment:
I think it would be better to replace the version in the path with a wildcard, as you'll otherwise have to keep changing this version, each time there's a new release of the `yajsw`.

Reply:
Yes, that's true. Unfortunately, wildcards will not work here. Therefore I will introduce a property for the whole yajsw lib name.